### PR TITLE
Add Prerequisites for OpenSuse users

### PR DESCRIPTION
**Pull request by henriquemeira**

### CODE REVIEW

File: INSTALL [hunk 1]
```diff
@@ -14,6 +14,13 @@ You need autotools to build the code from source:
 
     apt-get update
     apt-get install -y autotools
+    
+    For OpenSuse
```
Comment (jubalh) [inline INSTALL]: its written: openSUSE ;)

Comment (henriquemeira) [inline INSTALL]: Should I send you another pull request?

Comment (kernc) [inline INSTALL]: That's fine. But could you unindent the block a level? Indentation, in this context, signifies literal code (bash) environment, not contents hierarchy. You could as well make the diff a part of the same code block, e.g.
```bash
# on Debian-based distros
apt-get ...

# on openSUSE
zypper ...
```

Comment (henriquemeira) [inline INSTALL]: The "autotools" package don't exists on openSUSE repositories, so, the description: "You need autotools to build..." don't make sense. Can I change to: "You need some tools to build ...", or some thing similar? Can you suggest?

```
Prerequisites
-------------

You need some tools to build the code from source:

    # On Debian-based distros
    apt-get update
    apt-get install -y autotools
    
    # On openSUSE
    zypper install automake
    zypper install gcc-c++

```


p.s.: sorry for my poor english  ;)

Comment (kernc) [inline INSTALL]: Technically, automake is a part of autotools, so both are fine.

**Comment by kernc** [comment]
Thanks!
